### PR TITLE
Common: update RTF page

### PR DESCRIPTION
**Pull request by rmackay9**
These changes can be a big controversial but I've tried to update the page as best I can including:

- Refreshed links for product changes for Event38, CUAV, FlyDragonDroneTech
- Removed duplicate link for MakeFlyEasy Fighter
- removed links for AionRobotics, WattsInnovations
- Alphabetised ordering

Also I added BathyDrone to the non-partners section

I've tested this locally and it looks OK to me

### CODE REVIEW

File: common/source/docs/common-rtf.rst [hunk 1]
```diff
@@ -90,19 +73,26 @@ Subs from Partners
 
 * `Blue Robotics - BlueROV2 <https://bluerobotics.com/store/rov/bluerov2/>`__
 
-
 Vehicles from Non-Partners
 ==========================
 
+* `AION ROBOTICS - R6 UGV <https://www.aionrobotics.com/r6>`__
```
Comment (Hwurzburg) [inline common/source/docs/common-rtf.rst:79]:
```suggestion
* `AION ROBOTICS <https://www.aionrobotics.com>`__
```
no direct product links on the site...maybe should remove entirely

**Review by Hwurzburg — DISMISSED** [review]

File: common/source/docs/common-rtf.rst (hunk 1 above)
Comment (rmackay9) [reply common/source/docs/common-rtf.rst]: yup, good idea, removed.

**Comment by rmackay9** [comment]
@Hwurzburg let me make a couple more changes before we merge..

File: common/source/docs/common-rtf.rst [hunk 2]
```diff
@@ -90,19 +84,23 @@ Subs from Partners
...
 Vehicles from Non-Partners
 ==========================
 
 * `Aton <https://traxxas.com/products/models/heli/Aton-Plus>`__ and `Aton-Plus from traxxas <https://traxxas.com/products/models/heli/Aton-Plus>`__ (firmware loaded using an SD Card)
-* DRONEE  Easy to Use Mapping Plane Drone `DRONEE PLANE <https://dronee.aero/pages/droneeplane>`__
+* `BathyDrone mapping boats <https://www.bathydrone-usv.com/>`__
+* `DRONEE Easy to Use Mapping Plane <https://dronee.aero/pages/droneeplane>`__
 * `MotoDoro Farm Mapper (Plane) <https://motodoro.com/blog/detail/00005-farm-mapper-vtol.html>`__
 * `SkyRocket - Journey <http://sky-viper.com/journey/>`__
+* `TT Robotix - CX-180 ICEMAN Helicopter <http://www.ttrobotix.com/products/detail/925.html>`__
```
Comment (rishabsingh3003) [inline common/source/docs/common-rtf.rst:95]: I think we would need to add these back to the Partners list?

**Review by rishabsingh3003 — DISMISSED** [review]

File: common/source/docs/common-rtf.rst (hunk 2 above)
Comment (rmackay9) [reply common/source/docs/common-rtf.rst]: yes, sorry, very careless of me.  Not enough coffee yet.  Thanks!

**Comment by rmackay9** [comment]
OK @Hwurzburg, After a few mistakes on my part I think this is ready now.

**Pull request merged**